examples/react-app/request.ts

```diff
@@ -17,19 +16,31 @@ const axiosInstance = axios.create({
 });
 
 // Add a request interceptor
-axios.interceptors.request.use(
-  function (config) {
-    // Do something before request is sent
-    return config;
-  },
-  function (error) {
-    // Do something with request error
-    return Promise.reject(error);
-  }
+axiosInstance.interceptors.request.use(
+    function (config) {
+        // Do something before request is sent
+        if (!config.url || !config.params) {
+            return config;
+        }
+
+        Object.entries(config.params).forEach(([key, value]) => {
```

Comment:
```suggestion
        Object.entries<any>(config.params).forEach(([key, value]) => {
```

@Corepex  To avoid that TypeScript error, please use generics.

Reply:
@7nohe sorry didn't see the change. Will do the merge in the upcoming days.